### ISSUE TITLE
Error while generating wizardlm weights: missing utils file in train_freeform.py

### ISSUE THREAD

**Issue by gabriead** [nlpxucan/WizardLM#22]
Hi guys,

I followed the instructions to generate the weights from source. There is no file named "utils" imported in line 25 in train_freeform.py. Please provide that in the repo.

**Comment by NamburiSrinath** [comment]
Yes, how can we recover the original weights? Is there any update on this issue yet

I changed  the following line in `weight_diff_wizard.py `

`from train import smart_tokenizer_and_embedding_resize`
to 
`from train_freeform import smart_tokenizer_and_embedding_resize`

and then I am facing this issue.

**Comment by yatesdr** [comment]
You have to install lm-sys/Fastchat and rename

**Comment by ZhengYangCS** [comment]
# utils.py

import json
def jload(path):
    fr = open(path, 'r')
    d = json.load(fr)
    return d
